build.gradle

```diff
@@ -6,19 +6,18 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.0"
-        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:3.2.4"
+        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:$webdriverBinariesVersion"
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:3.4.6"
         classpath 'org.grails.plugins:quartz:2.0.13'
     }
 }
+
 plugins {
- //   id 'org.springframework.boot' version '1.4.2.RELEASE'
-    // Add Git properties plugin.
-    id 'com.gorylenko.gradle-git-properties' version '1.4.17'
+    id "com.gorylenko.gradle-git-properties" version "2.4.1"
 }
 
 
-version "2.4"
+version "2.4.1-SNAPSHOT"
```

Comment:
We should probably bump the minor or major version.

Reply:
Agreed. Will bump up the minor version `2.5.0` as I've done with some other apps